experimental/packages/exporter-trace-otlp-http/src/platform/browser/OTLPTraceExporter.ts

```diff
@@ -53,7 +53,7 @@ export class OTLPTraceExporter
     return typeof config.url === 'string'
       ? config.url
       : getEnv().OTEL_EXPORTER_OTLP_TRACES_ENDPOINT.length > 0
-        ? appendRootPathToUrlIfNeeded(getEnv().OTEL_EXPORTER_OTLP_TRACES_ENDPOINT, DEFAULT_COLLECTOR_RESOURCE_PATH)
```

Comment:
you removed the appending on the signal specific path when there is none ? not sure to see why ?

Reply:
> For the per-signal variables (`OTEL_EXPORTER_OTLP_<signal>_ENDPOINT`), the URL MUST be used as-is without any modification. The only exception is that if an URL contains no path part, the root path `/` MUST be used (see [Example 2](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md#example-2)).

there might be a path part that does not end in `/v1/traces` or `/v1/metrics`. With the previous implementation, it would append `/` to that path, which can result in `404`s, see #3269

Reply:
(from the exporter spec: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md#endpoint-urls-for-otlphttp)